docs/pages/archived/managed-workflow.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Managed workflow
+---
+
+> TL;DR: "Managed workflow" is now just referred to as "Using Expo Prebuild".
+
+Up until SDK 41, the terms **managed workflow** and **bare workflow** had a completely different set of limitations than they do today, and developers had to choose which set of features they wanted to use.
+
+A **managed workflow** project _had_ the following drawbacks:
+
+- Could only develop in the [Expo Go][expo-go] app.
+- No ability to use any custom native code.
+- Large binary sizes as apps were all embedded in a [shell app][shell-app].
+- Had to use the classic Expo build service.
+
+A **bare workflow** project _had_ the following drawbacks:
+
+- No ability to use Expo services (notifications, OTA updates, builds, submissions).
+- Could not use the classic build service.
+- Could not run in the [Expo Go][expo-go] app.
+- No [Prebuild][prebuild]-like functionality for generating native projects.
+- Substantially harder to upgrade, build, test, and distribute.
+
+Users would start in the **managed workflow** and eventually **eject** to the **bare workflow** by running `expo eject` (formerly `exp detach`), a now deprecated command that would perform some archaic native code generation that often didn't work. It was nearly impossible to go from bare workflow back into managed workflow.
+
+## Replacements
+
+By reducing **managed workflow** down to **using Expo Prebuild** and making all of our other tools flexible enough to use with any React Native project, Expo no longer has any concept of _custom workflows_ that create lock-in and generate development caveats. You can think of all projects generated with `npx create-expo-app` as React Native apps that have access to any of the tools offered by the Expo team, this includes Expo Prebuild, Expo Native Modules API, Expo SDK, Expo CLI, EAS CLI, EAS Update, EAS Build, EAS Submit, Expo Go, Config Plugins, Dev Clients, etc.
```

Comment:
```suggestion
By reducing **managed workflow** down to **using Expo Prebuild** and making all of our other tools flexible enough to use with any React Native project, Expo no longer has any concept of _custom workflows_ that create lock-in and generate development caveats. Instead, you can think of all projects generated with `npx create-expo-app` as React Native apps that have access to any of the tools offered by the Expo team. This includes Expo Prebuild, Expo Native Modules API, Expo SDK, Expo CLI, EAS CLI, EAS Update, EAS Build, EAS Submit, Expo Go, Config Plugins, Dev Clients, and so on.
```

- A very long sentence that was hard to read so let's break it down into two parts.
- Also, as per the Writing Style Guide, let's avoid using [Latin abbreviations](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations) such as `etc.`